daemon/config/config.go

```diff
@@ -144,6 +144,7 @@ type CommonConfig struct {
 	AuthorizationPlugins  []string                  `json:"authorization-plugins,omitempty"` // AuthorizationPlugins holds list of authorization plugins
 	AutoRestart           bool                      `json:"-"`
 	Context               map[string][]string       `json:"-"`
+	DefaultCapabilities   []string                  `json:"capabilities,omitempty"`
```

Comment:
```suggestion
	DefaultCapabilities   []string                  `json:"default-capabilities,omitempty"`
```

Actually thinking where we use the JSON option, I _think_ this is used when reloading the config (in which case the active config is logged https://github.com/moby/moby/pull/36019)


This option should also be moved to the `Config` type in `config_unix`, because this is Linux-specific, and doesn't apply to Windows daemons; https://github.com/moby/moby/blob/e6c1820ef5de8c198b4ddec74440a7ea7b331194/daemon/config/config_unix.go#L26-L33